assets/js/mixins/formatter.js

```diff
@@ -14,9 +14,17 @@ const ENERGY_PRICE_IN_SUBUNIT = {
   USD: "¢", // US cent
 };
 
+export const WATT_FORMAT = Object.freeze({
+  WATT: "W",
+  KW: "kW",
+  MW: "MW",
+  AUTO: "",
+});
```

Comment:
I'd change `WATT_FORMAT` to  `POWER_UNIT` or even consider exporting them flat: `POWER_W`, `POWER_KW`, `POWER_MW` and `POWER_AUTO`.

Reply:
Sorry, I think I was unclear here or confusing. I wouldn't change both. 

I saw the `POWER_[W|KW|MW|AUTO]` option as an alternative to the enum/object style. If we export as an object theres no need for the `POWER_` prefix on all values. I was thinking about exporting all values as single consts when i mentioned it:

```
export const POWER_W = ...
export const POWER_KW = ...
...
```

but doing 

```
export const POWER_UNIT = Object.freeze({
  W: "W",
  KW: "kW",
  ...
});
```

is also fine.

Reply:
No problem. In this case I'd like to stay with the enum and rename the values again.